contrib/highlight.js/Configuration

```diff
@@ -0,0 +1,24 @@
+Selected languages for custom package:
+
+Bash
+C++
+CSS
+Diff
+HTML, XML
+HTTP
+Ini
+JSON
+Java
+JavaScript
+Makefile
+Nginx
+PHP
+Perl
+Python
+Ruby
+SQL
+
```

Comment:
This is more of a reminder which configuration options I chose on the Web site. This is not a formalized configuration file.

On the Web site, there is an upper part of the list (commonly used options) & a bottom part of the list (the long tail options).